src/components/vm/nics/nicBody.jsx

```diff
@@ -97,6 +97,7 @@ export const NetworkTypeAndSourceRow = ({ idPrefix, onValueChanged, dialogValues
             {
                 name: 'direct',
                 desc: 'Direct attachment',
+                detailHeadline: _("This is the recommended config for high performance or enhanced security. In the default 'vepa' mode, a guest virtual machine's packets are always sent via the physical interface. Communication between guest virtual machines, or between a guest virtual machine and the host is not possible unless the physical interface is connected to a VEPA-capable network switch. Other modes are available; consult the libvirt documentation for further information."),
```

Comment:
This is rather  wordy but it definitely explains the option. @martinpitt do you want to have a second word here?

Reply:
Is it possible to use markup to render "vepa" in monospace, or to include a nice link directly to https://libvirt.org/formatdomain.html#direct-attachment-to-physical-interface?

Reply:
Should we follow libvirt docs and say ' all VMs' packets are sent to the external bridge' instead of the current wording?
I also just checked it live in the popover and it's incredibly long, we need to shorten it.

Reply:
How about:

> In the default 'vepa' mode, switching is offloaded to the ethernet switch. If the switch is not VEPA-capable, communication between guest virtual machines, or between a guests and the host is not possible. See the linked documentation for information about the other modes.

Reply:
I like it! I am only hesitant about being so verbose about existanse of 'other modes' since we offer the option to change the mode. 

All in all - I would keep the whole suggested text - but for the last sentence. 
Instead of that I would rather use a plain 'more info' link as I had in the example branch I sent above.

Reply:
Sounds good